mlrun/api/api/endpoints/healthz.py

```diff
@@ -1,76 +1,15 @@
 from fastapi import APIRouter
 
-from mlrun.api.utils.clients import nuclio
-from mlrun.config import config, default_config
-from mlrun.runtimes.utils import resolve_mpijob_crd_version
-from mlrun.utils import logger
+import mlrun.api.crud.client_spec
+import mlrun.api.schemas
 
 router = APIRouter()
 
 
-@router.get("/healthz")
+# TODO: From 0.7.0 client uses the /client-spec endpoint,
+#  when this is the oldest relevant client, remove this logic from the healthz endpoint
```

Comment:
I meant this comment to be above the `get_client_spec` call
we're not going to remove the healthz endpoint, we just won't need it to return the config anymore